packages/aptos-nextra-components/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@aptos-labs/nextra-components",
+  "version": "0.0.0",
+  "description": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "main": "lib/index.cjs",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "type": "module",
+  "files": [
+    "lib",
+    "dist"
+  ],
+  "scripts": {
+    "ts-types": "tsc",
+    "build:dev": "rimraf lib && tsup --watch",
+    "build:prod": "rimraf dist && env NODE_ENV=production tsup",
+    "build:watch": "tsup --watch",
```

Comment:
What's rimraf do?

Reply:
rm -rf :)

Reply:
It's basically `rm -rf`, but cross-platform (aka Windows support)